### PR TITLE
commented the code for elastic search from varchar field template

### PR DESCRIPTION
**Pull request by harjeet-nyt**

### CODE REVIEW

**Comment by harjeet-nyt** [comment]
@ash-nyt @jcora-nyt Please review the change to make unified_search by default false. I have commented all the lines related to elastic search instead of making it "false".

File: res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig [hunk 1]
```diff
@@ -17,12 +17,12 @@ $dictionary["{{ moduleBean }}"]["fields"]["{{ field }}"] = array(
     'comment' => '',
     'help' => '',
     // Sugar search features
-    'unified_search' => true,
-    'full_text_search' => array(
-        'enabled' => true,
-        'searchable' => true,
-        'boost' => 1.00
-    ),
+    //'unified_search' => false,
```
Comment (ash-nyt) [inline res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig]: Why did you comment this out?

File: res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig [hunk 2]
```diff
@@ -17,12 +17,12 @@ $dictionary["{{ moduleBean }}"]["fields"]["{{ field }}"] = array(
...
+    //'full_text_search' => array(
+    //    'enabled' => false,
+    //    'searchable' => true,
```
Comment (ash-nyt) [inline res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig]: Also why is this true?

**Review by ash-nyt — CHANGES_REQUESTED** [review]

File: res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig (hunk 1 above)
Comment (harjeet-nyt) [reply res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig]: I commented the lines as I thought if we need to enable the search feature we can uncomment the lines.

File: res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig (hunk 2 above)
Comment (harjeet-nyt) [reply res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig]: It was already there for search feature enabled fields. If we do not want to enable search feature we can remove these lines altogether. That is why I commented them.

Comment (jcora-nyt) [reply res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig]: @harjeet-nyt, remove the commented lines except for `unified_search` that the value should be false by default.

Comment (harjeet-nyt) [reply res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig]: @jcora-nyt, made the change.

**Review by jcora-nyt — APPROVED** [review]
Looks good. Merging.

**Pull request merged**